tests/test_mixins.py

```diff
@@ -4,72 +4,74 @@
 
 import dpnp as inp
 
+from .helper import get_float_dtypes
+
 
 class TestMatMul(unittest.TestCase):
     def test_matmul(self):
         array_data = [1.0, 2.0, 3.0, 4.0]
         size = 2
 
-        # DPNP
-        array1 = inp.reshape(
-            inp.array(array_data, dtype=inp.float64), (size, size)
-        )
-        array2 = inp.reshape(
-            inp.array(array_data, dtype=inp.float64), (size, size)
-        )
-        result = inp.matmul(array1, array2)
-        # print(result)
-
-        # original
-        array_1 = numpy.array(array_data, dtype=numpy.float64).reshape(
-            (size, size)
-        )
-        array_2 = numpy.array(array_data, dtype=numpy.float64).reshape(
-            (size, size)
-        )
-        expected = numpy.matmul(array_1, array_2)
-        # print(expected)
-
-        # passed
-        numpy.testing.assert_array_equal(expected, result)
-        # still failed
-        # self.assertEqual(expected, result)
+        for dtype in get_float_dtypes():
```

Comment:
Wouldn't it be better to use pytest parametrize instead of nested loop? Or is there any reason why it can't be implemented here?